packages/popover/src/Popover.types.ts

```diff
@@ -50,6 +51,12 @@ type Justify = (typeof Justify)[keyof typeof Justify];
 
 export { Justify };
 
+export type ExtendedPlacement =
+  | Placement
```

Comment:
Are we still supporting `Justify.Fit`? Should that be marked `@deprecated`?

Reply:
If yes, then this screenshot is wrong, if no then we should mark it `@deprecated`
![Screenshot 2024-09-13 at 12 37 12 PM](https://github.com/user-attachments/assets/4efc3ab1-f77d-4ae4-bb03-0662ebb85adf)